packages/workspace/src/generators/ci-workflow/__snapshots__/ci-workflow.spec.ts.snap

```diff
@@ -30,11 +30,7 @@ jobs:
       - bash: az devops configure --defaults organization=$(System.TeamFoundationCollectionUri) project=$(System.TeamProject)
         displayName: 'Set default Azure DevOps organization and project'
       # Get last successfull commit from Azure Devops CLI
-      - displayName: 'Get last successful commit SHA'
-        condition: ne(variables['Build.Reason'], 'PullRequest')
-        env:
-          AZURE_DEVOPS_EXT_PAT: $(System.AccessToken)
-        bash: |
+      - bash: |
```

Comment:
was informed that this actually does work if the repo is hosted in GitHub and uses ADO pipelines? 

unsure how that magically works, but should still default to work in both platforms if possible